.idea/misc.xml

```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Hmm I don't remember changing this. Not sure what the after effects would be.

Reply:
I think it specificies the Java version. Please update your Java to Java 8 if you haven't already.

Reply:
Will do